go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
1. Does "graph configuration" means the V2 proto configuration python API?
1. V2 config python API only work when using the new pserver. I think we can just remove the python API for just less confusing, and in most cases people always config a "global" optimizer algorithm for all parameters(layers), and do specific configuration for only some of the parameters.

Do you agree with removing python API only?